packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1313,21 +1331,47 @@ export default class Meeting extends StatelessWebexPlugin {
    * @memberof Meeting
    */
   setUpLocusInfoSelfListener() {
-    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_MUTED_BY_OTHERS, (payload) => {
-      if (payload) {
+    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_LOCAL_UNMUTE_REQUIRED, () => {
+      if (this.audio) {
+        this.audio.handleServerLocalUnmuteRequired();
         Trigger.trigger(
           this,
           {
             file: 'meeting/index',
             function: 'setUpLocusInfoSelfListener'
           },
-          EVENT_TRIGGERS.MEETING_SELF_MUTED_BY_OTHERS,
+          EVENT_TRIGGERS.MEETING_SELF_UNMUTED_BY_OTHERS,
+          {}
+        );
+      }
+    });
+    this.locusInfo.on(LOCUSINFO.EVENTS.SELF_REMOTE_MUTE_STATUS_UPDATED, (payload) => {
+      if (payload) {
+        if (this.audio) {
+          this.audio.handleServerRemoteMuteUpdate(payload.muted, payload.unmuteAllowed);
+        }
+        Trigger.trigger(
+          this,
           {
-            payload
-          }
+            file: 'meeting/index',
+            function: 'setUpLocusInfoSelfListener'
+          },
+          payload.muted ? EVENT_TRIGGERS.MEETING_SELF_MUTED_BY_OTHERS : EVENT_TRIGGERS.MEETING_SELF_UNMUTED_BY_OTHERS,
+          payload.muted ? {unmuteAllowed: payload.unmuteAllowed} : {}
```

Comment:
better to no break the API and just pass the payload and not introduce the new "unmuted" event

Reply:
after some further discussion we agreed that new "unmuted" event is needed, but we should keep sending the whole payload for both muted and unmuted events and just highlight the unmuteAllowed field in the docs